schemas/user_agent.yml

```diff
@@ -14,7 +14,14 @@
       level: extended
       type: keyword
       description: >
-        Unparsed version of the user_agent.
+        Unparsed keyword version of the user_agent.
+      example: "Mozilla/5.0 (iPhone; CPU iPhone OS 12_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.0 Mobile/15E148 Safari/604.1"
+
+    - name: original.text
```

Comment:
@mbudge Thanks for making the naming adjustment requested by Nicolas.

The way to add `text` as a multi-field would be by modifying the definition of `user_agent.original`. You can see another example of doing it in [this other PR](https://github.com/elastic/ecs/pull/505/files?file-filters%5B%5D=.md&file-filters%5B%5D=.yml#diff-6fe1ba8148033c8c0ae4ec82c9427385R27-R29).

Currently multi-fields support is broken\* in the generator script, but this will be fixed prior to the next release, so you can simply focus on putting in place a multi-field definition like in this the linked PR.

\* Multi-fields currently don't show up in the sample templates, and are not displayed specifically in the documentation. However they don't break the build, so if you make the adjustment we can merge this, and fix multi-fields separately.